src/utils/ExternalLink.tsx

```diff
@@ -1,6 +1,9 @@
 import React from "react";
 
-export default (url, text) => (
+export const externalLink = (
```

Comment:
This should be capitalized/Pascal case since it's a component.

Reply:
Nevermind. It _should_ be a normal component but I see that the way it's being used [elsewhere](https://github.com/unchained-capital/caravan/pull/350/commits/051e8ca43bb8e147144a3468ea9f0bf876af1ce5#diff-a11ecbc05f5aaa822e0dde77b05c8db9f813ecbecc514b8b5880de4d2db5c9cdR33-R37) isn't jsx style. We don't need to address this now.